modules/oe_editorial_entity_version/oe_editorial_entity_version.install

```diff
@@ -24,6 +24,27 @@ function oe_editorial_entity_version_install(): void {
     $data = [
       'create_new_draft' => [
         'minor' => 'increase',
+        'check_values_changed' => '1',
```

Comment:
Do we not need to add an update path for this changes?

Reply:
I think we don't since the option does not need to be present and the code was not deployed so far.

Reply:
:+1: